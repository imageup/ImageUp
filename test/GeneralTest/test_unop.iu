function main() -> int
{
	bool: t = true;
	bool: b = !true;
	int: a = -1;
	float: f = -0.66;
	if (a != 2){
		prints("not equal work");
	}
	printb(b);
	printf(f);
	return 0;
}