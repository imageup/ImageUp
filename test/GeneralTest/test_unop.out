not equal work
0
-0.66
